books/system/doc/render-doc-base.lisp

```diff
@@ -236,16 +237,23 @@
 ; we decide to associate additional information with name, then we may have a
 ; more convincing reason to make this change.
 
-    (mv (list* (rendered-symbol name)
+    (mv (list ;list*
+               (rendered-symbol name)
                (rendered-symbol-lst parents)
                terminal
+               from
+               #|| EM: review later
                (if (equal from "ACL2 Sources")
```

Comment:
I took this out for simplicity, since in my test it bloated only about 0.6%
However, once the downstream rendering is working I will put it back in later.